### DISCUSSION TITLE
Show Reconnect Button

### DISCUSSION THREAD

**Discussion by SJLennon** [codefori/vscode-ibmi#477] [General]
I've used the "Force reconnect to system" toolbar icon several times and it has worked well.  I'm curious about why there is a "Show Reconnect Button" to make it appear.  Wouldn't it just be easier to always show the icon in the toolbar?
@worksofliam

**Comment by worksofliam** [comment]
Good point. I think I made it optional because it's not for everyone. Perhaps, if the user gets disconnected automatically we could change the user's config to then display the button from then on out.

**Reply by SJLennon** [reply]
Not a big issue.  I was thinking about adding it to the documentation, then wondered why it was an option.  It probably doesn't warrant any other action and I suppose most people don't lose connections.
@worksofliam
